src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -618,108 +618,118 @@ void MavlinkReceiver::handle_message_command_both(mavlink_message_t *msg, const
 
 	} else if (cmd_mavlink.command == MAV_CMD_DO_AUTOTUNE_ENABLE) {
 
-		bool has_module = true;
-		autotune_attitude_control_status_s status{};
-		_autotune_attitude_control_status_sub.copy(&status);
-
-		// if not busy enable via the parameter
-		// do not check the return value of the uORB copy above because the module
-		// starts publishing only when MC_AT_START is set
-		if (status.state == autotune_attitude_control_status_s::STATE_IDLE) {
-			vehicle_status_s vehicle_status{};
-			_vehicle_status_sub.copy(&vehicle_status);
+		// Check command validity
+		if ((int)roundf(vehicle_command.param1) != 1 || (int)roundf(vehicle_command.param2) != 0) {
+			result = vehicle_command_ack_s::VEHICLE_CMD_RESULT_TEMPORARILY_REJECTED;
```

Comment:
I wouldn't call this "temporarily" as it won't fix itself unless the code is changed. Probably more a case of "DENIED", I'd say.

Reply:
THanks! You're right.

```suggestion
			result = vehicle_command_ack_s::VEHICLE_CMD_RESULT_DENIED;
```